### ISSUE TITLE
sync committee membership API broken

### ISSUE THREAD

**Issue by kevinwucodes** [prysmaticlabs/prysm#10023]
🐞  Bug Report

Prysm v2.0.5

This bug report is a continuation from issue #9940.  Prior, it was possible to obtain the current sync committee, but now both the **current epoch** and **current epoch + 256** return errors.  

{"message":"Could not fetch beacon state using request: rpc error: code = Internal desc = Invalid state ID: slot cannot be in the future","code":500}

Example calls:

```
curl 'http://192.168.0.1:3500/eth/v1/beacon/states/finalized/sync_committees?epoch=85615' 
curl 'http://192.168.0.1:3500/eth/v1/beacon/states/finalized/sync_committees?epoch=85871' 
```

**Comment by potuz** [comment]
Thanks for this, yes, this is a regression and we are working on it, #10015 fixes this issue, we need to add a test still.

**Comment by potuz** [comment]
Fixed by #10015, please reopen this if you continue experiencing this next release.
